controllers/admin/AdminImportController.php

```diff
@@ -2094,19 +2756,30 @@ protected function productImportOne($info, $default_language_id, $id_lang, $forc
                         $image = new Image();
                         $image->id_product = (int)$product->id;
                         $image->position = Image::getHighestPosition($product->id) + 1;
-                        $image->cover = (!$key && !$product_has_images) ? true : false;
+                        $image->cover = (!$key && !$productHasImages) ? true : false;
                         $alt = $product->image_alt[$key];
                         if (strlen($alt) > 0) {
                             $image->legend = self::createMultiLangField($alt);
                         }
                         // file_exists doesn't work with HTTP protocol
-                        if (($field_error = $image->validateFields(UNFRIENDLY_ERROR, true)) === true &&
-                            ($lang_field_error = $image->validateFieldsLang(UNFRIENDLY_ERROR, true)) === true && $image->add()) {
+                        if (($fieldError = $image->validateFields(UNFRIENDLY_ERROR, true)) === true
+                            && ($langFieldError = $image->validateFieldsLang(UNFRIENDLY_ERROR, true)) === true
+                            && $image->add()
+                        ) {
                             // associate image to selected shops
                             $image->associateTo($shops);
-                            if (!AdminImportController::copyImg($product->id, $image->id, $url, 'products', !$regenerate)) {
+                            if (!AdminImportController::copyImg(
+                                $product->id,
+                                $image->id,
+                                $url,
+                                'products',
+                                !$regenerate
+                            )) {
                                 $image->delete();
-                                $this->warnings[] = sprintf($this->trans('Error copying image: %s', array(), 'Admin.Advparameters.Notification'), $url);
+                                $this->warnings[] = $this->trans('Error copying image: %s',
```

Comment:
Same comment about translation placeholder. Please use a variable name surrounded by `%%` an add it in the second parameter.